src/main/java/moriyashiine/aylyth/common/entity/attachment/PledgeState.java

```diff
@@ -1,9 +1,9 @@
-package moriyashiine.aylyth.common.attachment;
+package moriyashiine.aylyth.common.entity.attachment;
```

Comment:
This is a World attachment, not an entity data attachment. Move to `world.attachment`